<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSources</class>
 <widget class="QWidget" name="DataSources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Sources</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNewAnalysis">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>New Analysis...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDetails">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="dataSourcesListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
